tests/linker/ios/link sdk/Bug2096Test.cs

```diff
@@ -44,10 +44,12 @@ public void Bug2096_Aot ()
 			
 			var x = new D ();
 			a.OuterMethod<D> (x);
+#if !__MACOS__
 			// fails with 5.0.2 (5.1) when running on devices with
 			// Attempting to JIT compile method 'A:InnerMethod<D, long> (D,long)' while running with --aot-only.
 			if (Runtime.Arch == Arch.SIMULATOR)
 				Assert.Inconclusive ("only fails on devices");
+#endif
```

Comment:
I wonder if we could move this to a common name and do a noop  on mac and do assert in iOS and others.